contracts/Governance.sol

```diff
@@ -19,13 +19,13 @@ contract Governance is
         Governor("OUSD Governance")
         GovernorSettings(
             1, /* 1 block */
-            32727, /* 5 days */
-            5000000 * 1e18 /* 5 mio veOgv */
+            17280, /* ~3 days (86400 / 15) * 3 */
```

Comment:
These days the average block time is more in the order of 13 sec. See https://ycharts.com/indicators/ethereum_average_block_time
So that would give a slightly larger number of blocks: 86400 / 13 * 3 = 19938

The initial code had used a value of 13.2sec to get to 32727 blocks = 5 days.

Though the calculation for GovernorPreventLateQuorum below was using 15sec in the old code. So I guess it was inconsistent...

Anyway it's an approximation. I think what you have of using 15sec and consistently using it for both settings is good.

Reply:
> These days the average block time is more in the order of 13 sec. See https://ycharts.com/indicators/ethereum_average_block_time So that would give a slightly larger number of blocks: 86400 / 13 * 3 = 19938
> 
> The initial code had used a value of 13.2sec to get to 32727 blocks = 5 days.
> 
> Though the calculation for GovernorPreventLateQuorum below was using 15sec in the old code. So I guess it was inconsistent...
> 
> Anyway it's an approximation. I think what you have of using 15sec and consistently using it for both settings is good.

Exactly my thinking. I was tempted to go with the current block mine time but noticed 15 seconds being used previously so I standardised everything.